### PR TITLE
NERFS THE SHIT OUT OF C4+THROWING STARS BECAUSE ITS CHEESY AND SHAMEFUL

### PR DESCRIPTION
**Pull request by Cheridan**
Cleans up X4 code copypaste. 
Attaching C4/X4 to objects makes them less aerodynamic.

### CODE REVIEW

**Comment by Repukan** [comment]
This kills explosive stars

File: code/game/objects/items/weapons/grenades/plastic.dm [hunk 1]
```diff
@@ -78,6 +102,12 @@
 		src.target = AM
 		forceMove(null)	//Yep
 
+		if(istype(AM, /obj/item)) //your crappy throwing star can't fly so good with a giant brick of c4 on it.
+			var/obj/item/I = AM
+			I.throw_speed = max(1, (I.throw_speed - 3))
+			I.throw_range = max(1, (I.throw_range - 3))
+			I.embed_chance = 0
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/grenades/plastic.dm]: lame

**Comment by PKPenguin321** [comment]
very deceiving and lame

**Comment by Cheridan** [comment]
There is nothing deceptive about anything here.

**Comment by PKPenguin321** [comment]
this is deliberately about throwing stars and spears with C4, yet the title is "C4 tweak" and the desc makes only a vague mention of aerodynamics

**Comment by PKPenguin321** [comment]
sneak-thief

**Comment by Cheridan** [comment]
it *is* a C4 tweak

**Comment by PKPenguin321** [comment]
dishonorable

**Comment by Cheridan** [comment]
have i given you satisfaction @PKPenguin321

**Comment by Armhulen** [comment]
no, close this

**Comment by jughu** [comment]
High tech explosive ninja stars when

**Comment by kevinz000** [comment]
cheridan is a hero

**Comment by Cheridan** [comment]
dynamite

**Comment by PKPenguin321** [comment]
Redemption has been found

File: code/game/objects/items/weapons/grenades/plastic.dm [hunk 2]
```diff
@@ -78,6 +102,12 @@
 		src.target = AM
 		forceMove(null)	//Yep
 
+		if(istype(AM, /obj/item)) //your crappy throwing star can't fly so good with a giant brick of c4 on it.
+			var/obj/item/I = AM
```
Comment (Razharas) [inline code/game/objects/items/weapons/grenades/plastic.dm]: This whole thing could really use some encapsulation
Like Attach() proc for objects or smth liek dis

**Review by Razharas — COMMENTED** [review]
I told about this stuff being deadly back in the first embedding pr
They said none will use it

**Comment by Aloraydrel** [comment]
No fun allooooowed

File: code/game/objects/items/weapons/grenades/plastic.dm (hunk 2 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/grenades/plastic.dm]: >They said none will use it

they (they being remie who made embedding) also said it was intentional emergent gameplay

**Comment by Repukan** [comment]
Hugbox

**Comment by Supermichael777** [comment]
i ded

**Comment by AdamElTablawy** [comment]
You can still do this with bola. And it's cheaper.

**Comment by CPTANT** [comment]
Can't say I ever saw this used.

**Comment by Cheridan** [comment]
@CPTANT You don't play enough, then.
@AdamElTablawy Still harder to use after this since it throws with less speed and distance.

**Comment by optimumtact** [comment]
This is sensible nerf in line with what would happen if you attached a heavy object to a throwing star.

However it still leaves room for someone to pull off a sweet move with this.

**Pull request merged**